t/admin/services.t

```diff
@@ -26,12 +26,12 @@ run_tests;
 
 __DATA__
 
-=== TEST 1: set service(id: 1)
+=== TEST 1: set service(id: 5eeb3dc90f747328b2930b0b)
```

Comment:
that is not a good idea to change the old test cases.

you can add new test cases for your new code.

Reply:
OK, I add a new test case

Reply:
Add t/admin/services-string-id.t